app/Models/V2/Forms/Application.php

```diff
@@ -13,14 +13,12 @@
 use Illuminate\Database\Eloquent\Relations\HasMany;
 use Illuminate\Database\Eloquent\Relations\HasOne;
 use Illuminate\Database\Eloquent\SoftDeletes;
-use Laravel\Scout\Searchable;
 
 class Application extends Model
 {
     use HasFactory;
     use HasUuid;
     use SoftDeletes;
-    use Searchable;
```

Comment:
🎨 This approach is fine and you can leave as-is if you prefer it this way, but I'd probably have made the new method with the name `search` for brevity (Application::searchApplications is a little duplicative). With this trait being removed, there shouldn't be any problem with that name.